iModelJsNodeAddon/IModelJsNative.cpp

```diff
@@ -2085,12 +2084,18 @@ struct NativeDgnDb : BeObjectWrap<NativeDgnDb>, SQLiteOps<DgnDb>
             options.m_customSchemaContext = NativeECSchemaXmlContext::Unwrap(maybeEcSchemaContextVal.As<Napi::Object>())->GetContext();
             }
 
+        LastErrorListener lastError(db);
         DbResult result = JsInterop::ImportSchemas(db, schemaFileNames, SchemaSourceType::File, options);
-
-        return Napi::Number::New(Env(), (int)result);
+        if (DbResult::BE_SQLITE_OK != result) {
+            if (lastError.HasError()) {
+                BeNapi::ThrowJsException(info.Env(), lastError.GetLastError().c_str(), (int) result);
+            } else {
+                BeNapi::ThrowJsException(info.Env(), "Failed to import schemas", (int) result);
+            }
+        }

```

Comment:
Braces should be correctly indented. In outermost it's starting braces on the next line, while code in side block has it on same line.

Reply:
Fixed the indentation